last correction after porting in kali 2020

giacomomangelli opened the pull request:
addiotion of a line for convertion of byte in string and corrected the suricata if statement

giacomomangelli committed:
Create container.py

addiotion of a line for convertion of byte in string

giacomomangelli committed:
Update container.py

giacomomangelli committed:
Update container.py

Meliodas123 commented on container.py:
```diff
@@ -0,0 +1,23 @@
+def is_ips():
+    """quick check to see if suricata is running"""
+
+    if psef(b'suricata'):
```
addion of b for converting in byte

giacomomangelli committed:
W4SP_WEBAPP.PY LINE 213

def is_ips():
    """quick check to see if suricata is running"""

    if psef(b'suricata'):
        return 'ok',200
    else:

giacomomangelli committed:
Delete CONTAINER.PY LINE 228
